app/src/main/res/layout/checkbox_left_preference.xml

```diff
@@ -13,6 +13,7 @@
     android:gravity="center_vertical"
     android:paddingStart="8dp"
     android:paddingBottom="8dp"
+    android:layout_marginBottom="8dp"
```

Comment:
I would suggest adding paddingTop=8dp instead, so that when you press it and the ripple effect takes place, the text and checkbox look centered in the button